src/navigator-card-body.jsx

```diff
@@ -292,80 +292,11 @@ export const NavigatorCardBody = ({
         }
     };
 
-    const handleClick = (ev, file) => {
```

Comment:
Please add to the commit message why this helps with skipping pointless re-drawing.

Reply:
This method is only used in `Item` component` so it belongs there.